index.html

```diff
@@ -7,7 +7,184 @@
     <link
       rel="stylesheet"
       href="node_modules/bootstrap/dist/css/bootstrap.min.css" />
-    <title>Document</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
```

Comment:
You should not be using bootstrap CDN for styles

Reply:
Removed CDN from head on all html.